### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by parteekcoder**
issue #2126 

Updated README.md to help contributors

### CODE REVIEW

**Comment by raclim** [comment]
Hi @parteekcoder, thanks for working on this. We currently do have [documentation for setting up your development environment](https://github.com/processing/p5.js-web-editor/blob/develop/contributor_docs/installation.md) within our [contributor docs](https://github.com/processing/p5.js-web-editor/tree/develop/contributor_docs) that includes these instructions.

I do think these docs could be harder to find, and having it noted somewhere in the README will make it more accessible. Maybe instead of the installation instructions, you can add a link to the contributor docs and some contextual information about them. Let me know what you think!

**Comment by parteekcoder** [comment]
> Hi @parteekcoder, thanks for working on this. We currently do have [documentation for setting up your development environment](https://github.com/processing/p5.js-web-editor/blob/develop/contributor_docs/installation.md) within our [contributor docs](https://github.com/processing/p5.js-web-editor/tree/develop/contributor_docs) that includes these instructions.
> 
> I do think these docs could be harder to find, and having it noted somewhere in the README will make it more accessible. Maybe instead of the installation instructions, you can add a link to the contributor docs and some contextual information about them. Let me know what you think!

@raclim so shall I remove these installation steps and just mention the documenation link in the README.md?

**Comment by parteekcoder** [comment]
hey @raclim now you can check and please tell if you like to mention anything else also

And please @raclim can you please guide how can I contribute to p5.js-web-editor project in GSOC this year

File: README.md [hunk 1]
```diff
@@ -16,6 +16,12 @@ Learn more about [our community](https://p5js.org/community/) and read our commu
 
 Make your first sketch in the [p5.js Editor](https://editor.p5js.org/)! Learn more about sketching with p5.js on the [Get Started](https://p5js.org/get-started/) and find everything you can do in the [Reference](https://p5js.org/reference/). You can also look at [examples](https://editor.p5js.org/p5/sketches) and remix them in the p5.js Editor.
 
+## Setting Development Environment
```
Comment (raclim) [inline README.md]: I might update this to be "Setting Up the Development Environment"!

File: README.md [hunk 2]
```diff
@@ -16,6 +16,12 @@ Learn more about [our community](https://p5js.org/community/) and read our commu
...
+
+- Refer here https://github.com/processing/p5.js-web-editor/blob/develop/contributor_docs/installation.md
```
Comment (raclim) [inline README.md]: I might also update this line to be a bit more descriptive and the URL to be linked! For ex. (Refer to [this documentation for setting up your environment](url-to-docs)).

**Review by raclim — COMMENTED** [review]
Just updated with some comments! Once these are done, I think it'll be ready to go!

**Comment by parteekcoder** [comment]
now @raclim @catarak @davepagurek  you can review I updated this PR as you told

**Review by raclim — APPROVED** [review]

**Comment by parteekcoder** [comment]
hey @raclim thanks for merging,
I want to ask that I want to contribute in p5web-editor in the project "Adding Test Coverage", but as I see the project list on there only the p5 projects at high priority 
can you please guide me with this

**Comment by raclim** [comment]
Hi @parteekcoder! 

The projects labelled as "High Priority" only apply to the ones in the p5.js Github repository. The projects listed for the p5.js Web Editor do not have any designated priority, so please feel free to select any that speak to you the most. For this year's Google Summer of Code, we advise potential contributors to have a drafted proposal to receive the best feedback. I'll try to have an updated post for this outlining these steps later today!

**Comment by parteekcoder** [comment]
yeah but first all the high priority projects are assigned by GSoC admin then only the low ones , processing organization have many projects for each repository that's i just want to ask that is this project "Adding Test Coverage" or any other project in p5web-editor has the priority

**Comment by raclim** [comment]
All of the projects listed for the p5.js Web Editor are ones that we would consider highly needed or "high priority", which is why there are no designated labels for them. Adding increased test coverage is a project we'd love further development on, but it does not hold a higher or lower priority compared to the other web editor projects. 

Thus, please feel free to follow through with drafting a proposal around it if it's something you're interested in!

**Pull request merged**